Package.swift

```diff
@@ -56,7 +54,7 @@ let package = Package(
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
         .target(
             name: "Mini",
-            dependencies: ["RxSwift", "NIOConcurrencyHelpers", "RxOptional"]
+            dependencies: ["NIOConcurrencyHelpers"]
```

Comment:
```suggestion
            dependencies: [.product(package: "swift-nio", package: "NIOConcurrencyHelpers"]
```

Reply:
But again, if you're not using atomics anymore from swift-nio, you can delete the dependency